django_register/base.py

```diff
@@ -71,6 +71,10 @@ def max_length(self):
         if self._key_to_class:
             return max(len(key) for key in self._key_to_class)
 
+    @property
+    def choices(self):
+        return [(k, k.replace("_", " ").title()) for k in self._key_to_class]
```

Comment:
Shouldn’t this use the `verbose_name` if available on the class?

Reply:
Indeed it should